config/application.travis.yml

```diff
@@ -0,0 +1,5 @@
+production:
+  production_url: ""
```

Comment:
I wouldn't name these like this, I would name them as deployment_url and deployment_user, then they would be the same for any environments. What do you think?